### DISCUSSION TITLE
Declaração de imposto de renda com conta no exterior

### DISCUSSION THREAD

**Discussion by danielracca** [DyegoCosta/trabalhando-remoto#178] [General]
Hey galera, comecei a trabalhar pra uma empresa no exterior e estou recebendo vou receber em depósito mensal numa conta que possuo na Irlanda. Minha pergunta é: Tem como eu manter esse meu dinheiro em € na Irlanda e declarar meus impostos aqui no Brasil de boa, pagando em R$ a % necessária dos impostos?

**Comment by vanessametonini** [comment]
Se você mora no Brasil e de alguma maneira transferir esse dinheiro pro Brasil vai ter que declarar. 
Quando vc faz a declaração do IR vc tem que pegar o extrato de cada conta de cada banco que vc possuí e informar. Não sei qual a regra para contas no exterior, teria que verificar com a próprio Receita Federal.
Se você não mora no Brasil, vc tem que fazer a saída do país, que aí acredito que não precisa se preocupar mais com a declaração.

**Comment by lorranpalmeira** [comment]
Alguém sabe o valor que tem que pagar de IR referente ao salário recebido do exterior?
Outra Dúvida.
Se não trazer parte do dinheiro para o Brasil deixando no paypal ou enviando para os EUAs, para conta de investimento, esse dinheiro caso precise declarar é tributado imposto sob este valor?

**Comment by vanessametonini** [comment]
@lorranpalmeira, o imposto pode ser até 27% do valor.
Se vc usar programa Carnê Leão para declarar seu IR mensalmente, daí vc já consegue ver quanto que vai ser o imposto, pois ele gera uma DARF do valor informado para pagar. Ou vc pode deixar para pagar só na hora da declaração anual do IR, porém o risco de vir uma paulada de uma vez é grande.

Sobre o dinheiro no exterior (paypal ou banco de fora), eu acho que não precisa declarar. Porém seria bom procurar com algum contador/especialista.

**Comment by lorranpalmeira** [comment]
Muito obrigado @vanessametonini

**Comment by diegoddox** [comment]
Oi @vanessametonini,
quer dizer que, eu morando no Brasil e tendo um emprego em outro pais e o pagamento do meu salario e ocorrido em outro pais e não no Brasil eu não preciso fazer a declaracão?

**Comment by vanessametonini** [comment]
Você sempre tem que fazer a declaração do IR @diegoddox, é bom mesmo se for isento. A não ser que vc seja dependente de alguém (alguém declara por ti).
Acredito que vc não precisa declarar só quando vc der saída no país (daí vc n vai estar mais morando no BR)

**Comment by diegoddox** [comment]
Muito obrigado @vanessametonini <3

**Comment by mjacobus** [comment]
Pergunta, não tanto relacionada a trabalho remoto, mas a questão fiscal.

No caso do sujeito morar e pagar impostos de renda no exterior, como fica quamdo ele mudar para o Brasil, ou decidir comprar imóveis, carros, etc, com o dinheiro que juntou e pagou impostos fora... está ele sujeito a mais impostos sobre o dinheiro?

**Comment by danielracca** [comment]
Se você trabalha no exterior e paga imposto no exterior você não precisa pagar aqui.

A questão é, você só vai pagar impostos no exterior como pessoa física se estiver morando lá. Então se vc esta morando la, você tem que só fazer uma carta de saída do Brasil e pronto.

Se você esta morando aqui, e tem o dinheiro lá, você vai ter que pagar os impostos normal. Fazer o carnê leão mensal e pagar o proporcional pro governo.

Regards,
Daniel Racca

**Comment by mjacobus** [comment]
Obrigado @danielracca

**Comment by lbernardomaia** [comment]
Agora uma pergunta mais complicada, porem comum!

Vamos imaginar um cenario:

Moro fora do pais, porem nao declarei a saida definitava do Brasil porque volto ao brasil pelo menos um vez por ano (Nao configurando a situacao de nao residente porque eu sempre volto no periodo de 12 meses). O salario que eu receber no outro pais sera tributado no brasil mesmo que ele ja tenha sido tributado na fonte em outro pais?

Abs.

**Comment by fforbeck** [comment]
@lbernardomaia 
Não sou especialista no assunto, mas eu entendo que se você não fez declaração de saída, você ainda precisa declarar o IR no BR. Então será necessário declarar as contas que possui em outros países como **PF**.  Então a alíquota progressiva é aplicada sobre o valor que você tem fora do BR, o máximo é 27%.  O Brasil tem acordos de dupla tributação com diversos países. É válido verificar se há algum acordo com o país que você reside atualmente, caso exista, isso pode ajudar a reduzir os impostos. 

Uma outra forma de reduzir/adiar o pagamento de impostos é: abrir uma empresa no país que você reside atualmente (desde de que tenha algum benefício fiscal) ou então em um paraíso fiscal. Após isso, receba seu salário na conta da empresa, então você não precisa pagar os impostos de PF no brasil, só precisaria declarar que é o principal acionista da empresa X em tal país. Então você irá pagar impostos de PF no BR somente quando tirar grana da conta PJ para sua conta PF (seja no BR ou em outro país). Assim você posterga o pagamento de impostos usando uma empresa, caso não exista ou acordo de dupla tributação não seja interessante para você.

**Comment by lbernardomaia** [comment]
Entendi.

Obrigado @fforbeck .

**Comment by DanielleBorges** [comment]
@danielracca vc conseguiu declarar ser imposto de renda com conta no exterior? Pode me indicar seu contador? Eu acabei de me mudar para o Brasil e consegui manter meu emprego remote.  Também quero continuar recebendo na minha conta no exterior. Já conversei com vários contadores em SP e nenhum sabe me informar se isso é possível. Todos querem que eu traga o dinheiro para o Brasil, mas vou perder muito dinheiro abrindo uma conta PJ no banco ($), com as taxas do wire transfer($$$) e na conversão.

**Comment by pablo-sg-pacheco** [comment]
Bom, eu estou com a Contabilizei. É e bem barato, mas tem que abrir PJ. Mas
até nesse processo de abertura eles orientam. Muito mais barato que
qualquer contador:
http://contabilizei.com.br/

Eu ainda não cheguei a declarar, mas estou emitindo nota com beneficio
fiscal para o exterior, uma coisa assim.



2018-07-24 18:01 GMT-03:00 DanielleBorges <notifications@github.com>:

**Comment by renatonascalves** [comment]
@DanielleBorges Estava conversando com meu contador e se for declarar aqui no Brasil, parece que existe uma lei que diminui os impostos dos 6% para 3 e alguma coisa.

cc: @pablo-sg-pacheco

**Comment by DanielleBorges** [comment]
@pablo-sg-pacheco @renatonascalves, obrigada pela informação. Só para confirmar, vcs também recebem o salário em conta estrangeira e não transferem para o BR?

**Comment by pablo-sg-pacheco** [comment]
De nada :)

Eu recebo pelo BitWage e automaticamente o salário é transferido pra minha conta no BR

**Comment by renatonascalves** [comment]
@DanielleBorges Eu utilizo o Bitwage também. Cai direto na conta bancária da PJ.

**Comment by samiraguiar** [comment]
@DanielleBorges dá uma olhada no banco Inter, eles têm conta PJ gratuita.
Uma alternativa ao BitWage é a Payoneer, que é a que eu uso. As taxas são menores que o BitWage, mas o tempo entre você receber o dinheiro na conta do exterior e ele cair na sua conta do Brasil é maior (cerca de 3 a 8 dias).

**Comment by renatonascalves** [comment]
Verdade! E eu já vi casos que o Payoneer é bem burocrático, chegando a demorar mais de 30 dias para uma pessoa.

O Inter é barato, quase nenhuma taxa. Mas é uma dificuldade consegui ser aceito por eles. É tipo Nubank quando começaram...

**Comment by ppozeti** [comment]
Olá galera, tudo bem? Eu recentemente também voltei ao Brasil e continuei trabalhando remoto. As dicas de vocês foram muito boas! 
Quando cheguei, abri uma EIRELI, abri uma conta pessoa física no Banco Inter e logo depois de aprovado, abri uma conta pessoa jurídica, também no inter.
O processo da conta PF foi rápido, 2 dias eu já tinha a aprovação. Já o da PJ demorou mais um pouco, uma semana. Depois disso você tem que realizar um cadastro com eles na parte de cambio. Tudo é feito via e-mail.
No meu caso, recebo dinheiro dos EUA, então a empresa lá, realiza o pagamento para um banco em NY (intermediário do Inter) e logo após eles receberem, comunicam o Inter.
Então o inter me envia um e-mail avisando que receberam X dinheiro e a cotação do momento. É +- 6 centavos abaixo do $ comercial.
Após eu responder que aceito a cotação, é questão de minutos para o dinheiro aparecer na minha conta!
Detalhe: não pago absolutamente nada para manter a conta nem para receber o dinheiro! Super aprovado.

**Comment by pablo-sg-pacheco** [comment]
Legal Paulo.
Vc tentou abrir a conta PJ no Inter quando?

Eu tentei no começo desse ano e simplesmente não consegui.
Acabei abrindo uma conta PJ digital no Santander. Uma tal de SuperDigital.
É ruim infelizmente rs, mas pelo menos têm sido interessante por não ter
tarifas

2018-08-01 15:00 GMT-03:00 Paulo Pozeti <notifications@github.com>:

**Comment by ppozeti** [comment]
@pablo-sg-pacheco mês passado! Te deram algum feedback da razão?

**Comment by pablo-sg-pacheco** [comment]
Poxa, vou tentar de novo então.
Me falaram que era muita demanda.

2018-08-01 15:19 GMT-03:00 Paulo Pozeti <notifications@github.com>:

**Comment by its-felipe-almeida** [comment]
Paulo, qual banco intermediário você usa em NY? Como você chegou até ele? Foi recomendação do banco Inter? Atualmente eu recebo dinheiro através do Payoneer. A cotação acaba sendo igual a que você falou (-6 centavos), e eu recebo na minha conta pessoal.

**Comment by fforbeck** [comment]
Estou utilizando o Inter também e não precisa pagar para receber remessas do exterior. Tanto na conta PF quanto na conta PJ. Não sei se já falaram, mas é preciso abrir uma conta PF antes de abrir a PJ.
O spread é bem mais baixos que quase todos os outros bancos nacionais. Vale a pena. Você só não recebe um número IBAN. A remessa é enviada para o intermediário do banco Inter nos EUA e eles mandam a notificação para o Inter que existe uma ordem para a conta X. Aí o Inter entra em contato com você via email para fazer o fechamento. Bem susse. Vale a pena.

**Comment by clmadrazo** [comment]
Quanto tempo demora para receber desde que o cliente transfere pro banco intermediário gringo e que o dinheiro finalmente cai na sua conta?

**Comment by fforbeck** [comment]
Já consegui receber em 1 dia, mas geralmente 2. Depende muito do horário que foi feito o pagamento e o horário que você faz o fechamento. As vezes não da tempo de cair no mesmo dia a grana na sua conta. Uma coisa que agiliza o processo é você pedir para quem pagou te enviar o comprovante SWIFT da operação. Assim você envia esse comprovante para o Inter e eles conseguem pesquisar a transação. Não precisa ficar esperando os intermediários dar um ping.

**Comment by ppozeti** [comment]
@felipeaugusto1 São instruções do próprio Inter! Após o cadastro na parte de cambio deles, eles enviam um PDF com as instruções para o pagamento. Assim como mencionou o @fforbeck.

@samiraguiar Tem razão, eu também ficaria com o pé atrás. Mas preferi apostar nisso e não ter que botar meu pé em nenhum banco!

**Comment by yuriploc** [comment]
@ppozeti @fforbeck Se o Inter não tem custo nenhum pra PJ (manutenção, transferência internacional...) tá bem melhor do que a Husky, que cobra 4% fixo por transferência.

Vocês continuam usando o Inter? Vale a pena? Fazem o Carnê-Leão?

**Comment by fforbeck** [comment]
@yuriploc 
eu continuo utilizando o Inter. Ontem mesmo fechei uma ordem lá. 
USD comercial estava: R$ 3,79
Fechei em R$ 3,74.
Sem IOF, Sem Tarifa. Sem custo de manutenção da conta.

Foi na minha conta PJ, mas é a mesma coisa para PF.  Até agora não encontrei taxa melhor. Já utilizei Itaú, Bradesco, HSBC (nas antigas), corretora, bitwage, payoneer, etc.

**Comment by lcguida** [comment]
@fforbeck Mas você não tem que pagar SWIFT? Ou o seu pagador ?

**Comment by praguc** [comment]
Oi galera, tudo bem?

Eu vou começar trabalhar para uma empresa e receber dinheiro pelo Payoneer. Como nunca trabalhei para uma empresa do exterior, eu queria saber se alguém me pode dar algumas dicas.

Alguém transfira dinheiro para conta bancária no Brasil ou vocês gastam pelo cartão pré-pago? Se declara tudo que você recebe o só o que cai na conta do banco?

**Comment by samiraguiar** [comment]
@praguc vc declara tudo que receber, porque as taxas que vc paga pra transferir o dinheiro pro Brasil (como o spread no caso da Payoneer) são vistas como encargos da empresa.

Eu costumo usar o cartão pra fazer compras internacionais e quando estou em viagem porque dá pra sacar na moeda local. O cartão da Payoneer cobra no máximo 3,5% numa compra internacional (ou 0% se a moeda da compra for a mesma da moeda do cartão), o que é bem mais em conta do que pagar com um cartão tipo o Nubank no qual vc vai pagar a taxa da Payoneer pra mandar o dinheiro pro Brasil + 4% de spread do Nubank + 6,38% de IOF.

Já o saque vai depender muito do valor, porque quanto maior o valor, menores as taxas proporcionalmente falando. Isso porque a Payoneer cobra uma taxa fixa por saque (2,50 EUR pra cartão em euro e 3,15 US$ pra cartão em dólar), e os ATMs do mundo todo também cobram taxa por saque. Fiz um teste no Uruguai recentemente num ATM que cobrava 100 pesos por saque (que é até barato). Sacando 1.000 pesos, teria os 2,50 EUR da Payoneer + 100 pesos do ATM + taxa de 3,20% de conversão, que daria um total de quase 25% em taxas. Agora, sacando 10.000 pesos o total de taxas cairia pra 5,31%. Já pra trocar reais pra peso eu teria a taxa de 2% da Payoneer pra transferir pro Brasil + uns 6 ou 7% que as casas de câmbio estavam cobrando. Ou seja: o saque no exterior compensa se vc for sacar bastante dinheiro de uma só vez.

No Brasil não compensa sacar o dinheiro, compensa transferir. Fiz uma simulação aqui com um saque de R$ 10.000 de um cartão em euro num ATM que cobre R$ 10,00 por saque. Mesmo com esse valor alto (que provavelmente o ATM nem liberaria) as taxas ficam em 3,42%, que é consideravalmente maior do que a taxa máxima que a Payoneer cobra quando vc transfere o dinheiro pra conta do Brasil (2% pra euro e até 3% pra dólar).

E vale lembrar que o cartão tem uma anuidade de US$ 30, então não compensa pedir se vc não for usar.

**Comment by praguc** [comment]
@samiraguiar obrigado pelas dicas Samir. Então, se você gasta alguma coisa no cartão Payoneer (vamos dizer 1000 reais) e você retira na conta 4000 reais (independente das taxas etc), o que você declara no carne-leão e 4000 o 5000? Estou perguntando, porque o dinheiro não entra no Brasil?

Tbm, você recomendaria um banco? Eu tenho conta no Itaú, más pelo que vi no site do BACEN, de todos os bancos, Itaú tem taxa de câmbio pior.

**Comment by samiraguiar** [comment]
@praguc eu sempre declaro o que recebo.
Por exemplo, suponha que num mês eu receba US$ 1.000,00, transfira US$ 800 para a conta no Brasil e use US$ 200 pra gastar no cartão com compras no exterior. Eu declaro os US$ 1.000,00, não os US$ 800.

Sobre bancos, se vc vai usar a Payoneer não precisa se preocupar com câmbio porque o dinheiro é transferido diretamente em reais pra sua conta. A taxa de câmbio entraria em jogo se vc fosse usar o Itaú pra receber dinheiro, caso em que não precisaria da Payoneer. Mas já que vc vai fazer como pessoa física eu recomendaria o Nubank mesmo (mas vc pode usar qualquer conta que tiver porque a Payoneer transfere pra lá). Transfere dinheiro da Payoneer pro Nubank e declara no carnê-leão o valor que vc recebeu na Payoneer.

Em todo caso, dá uma lida nesse tópico mesmo e nos outros desse repositório que tem várias alternativas pra receber dinheiro do exterior. Creio que os meios mais utilizados sejam o Bitwage, o Inter e a Payoneer, mas tem outras formas também.

E só um adendo aqui, é possível que seu primeiro pagamento pela Payoneer de valor mais alto (mais de US$ 500, pelo menos) fique bloqueado. Nesse caso eles vão te pedir mais informações como contrato com a empresa e um site seu (pode ser seu perfil do LinkedIn), vão levar uns dias pra avaliar e depois desbloqueiam. Isso é só uma verificação pra evitar lavagem de dinheiro, novos pagamentos da mesma empresa não serão bloqueados.

**Comment by wmdebrito** [comment]
@DanielleBorges Conseguiu resolver a tua situacao? Estou na mesma situacao que vc, acabei de voltar para o Brasil e continuo trabalhando remoto e recebendo na minha conta no Canada, precisou abrir PJ? Pode me indicar um contador?

**Comment by DanielleBorges** [comment]
> @DanielleBorges Conseguiu resolver a tua situacao? Estou na mesma situacao que vc, acabei de voltar para o Brasil e continuo trabalhando remoto e recebendo na minha conta no Canada, precisou abrir PJ? Pode me indicar um contador?

Em 2018 eu fui até na receita federal buscando esclarecimentos e ninguém soube me dar uma resposta clara. Não sei se agora com essa onda de trabalho remoto alguma lei mudou. Decidi que era melhor pecar por excesso do que omissão. Abri uma PJ, utilizo o serviço do Bitwage e recebo aqui. Declaro a nota fiscal como exportação de serviços do valor recebido e pago por volta de 4% de imposto. Importante detalhe é que eu faço serviços administrativos, acho que alíquota é menor que desenvolvedores ou algo mais técnico. Fazia tempo que não entrava aqui, vi que tem outros caminhos além da Bitwage. Vou dar uma pesquisada no banco Inter e Payoneer como recomendado acima.

Contato do meu contador:


Rodrigo Amadio Pacheco
________________________________
Celular. (15)99105-6166 (Claro)
rodrigo@multifidam.com.br

**Comment by wmdebrito** [comment]
@DanielleBorges
> > @DanielleBorges Conseguiu resolver a tua situacao? Estou na mesma situacao que vc, acabei de voltar para o Brasil e continuo trabalhando remoto e recebendo na minha conta no Canada, precisou abrir PJ? Pode me indicar um contador?
> 
> Em 2018 eu fui até na receita federal buscando esclarecimentos e ninguém soube me dar uma resposta clara. Não sei se agora com essa onda de trabalho remoto alguma lei mudou. Decidi que era melhor pecar por excesso do que omissão. Abri uma PJ, utilizo o serviço do Bitwage e recebo aqui. Declaro a nota fiscal como exportação de serviços do valor recebido e pago por volta de 4% de imposto. Importante detalhe é que eu faço serviços administrativos, acho que alíquota é menor que desenvolvedores ou algo mais técnico. Fazia tempo que não entrava aqui, vi que tem outros caminhos além da Bitwage. Vou dar uma pesquisada no banco Inter e Payoneer como recomendado acima.
> 
> Contato do meu contador:
> 
> Rodrigo Amadio Pacheco
> 
> Celular. (15)99105-6166 (Claro)
> [rodrigo@multifidam.com.br](mailto:rodrigo@multifidam.com.br)

Muito Obrigado pela resposta. 
Acho que vou por esse caminho também. Eu queria mesmo continuar como PF, eu ainda tenho que ver como vai a questão do imposto no Canadá em relação ao que a minha empresa precisa fazer.
Eu trabalho numa empresa que faz transferências também, nosso foco são pequenas e medias empresas. Transferimos para o Brasil. Não é propaganda é só mais uma opção. Pode fazer uma simulação lá.
www.veem.com

**Comment by DanielleBorges** [comment]
Legal, vou pesquisar a Veem também.

Eu presto serviços para uma empresa Americana e tive que enviar o W-8BEN -
certificate of Foreign Status for USA tax withholding and reporting, pode
ter algo parecido no Canadá.

On Tue, 4 May 2021 at 20:06 Wagner Brito ***@***.***> wrote:

> @DanielleBorges <https://github.com/DanielleBorges>
>
> @DanielleBorges <https://github.com/DanielleBorges> Conseguiu resolver a
> tua situacao? Estou na mesma situacao que vc, acabei de voltar para o
> Brasil e continuo trabalhando remoto e recebendo na minha conta no Canada,
> precisou abrir PJ? Pode me indicar um contador?
>
> Em 2018 eu fui até na receita federal buscando esclarecimentos e ninguém
> soube me dar uma resposta clara. Não sei se agora com essa onda de trabalho
> remoto alguma lei mudou. Decidi que era melhor pecar por excesso do que
> omissão. Abri uma PJ, utilizo o serviço do Bitwage e recebo aqui. Declaro a
> nota fiscal como exportação de serviços do valor recebido e pago por volta
> de 4% de imposto. Importante detalhe é que eu faço serviços
> administrativos, acho que alíquota é menor que desenvolvedores ou algo mais
> técnico. Fazia tempo que não entrava aqui, vi que tem outros caminhos além
> da Bitwage. Vou dar uma pesquisada no banco Inter e Payoneer como
> recomendado acima.
>
> Contato do meu contador:
>
> Rodrigo Amadio Pacheco
>
> Celular. (15)99105-6166 (Claro)
> ***@***.***
>
> Muito Obrigado pela resposta.
> Acho que vou por esse caminho também. Eu queria mesmo continuar como PF,
> eu ainda tenho que ver como vai a questão do imposto no Canadá em relação
> ao que a minha empresa precisa fazer.
> Eu trabalho numa empresa que faz transferências também, nosso foco são
> pequenas e medias empresas. Transferimos para o Brasil. Não é propaganda é
> só mais uma opção. Pode fazer uma simulação lá.
> www.veem.com
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/DyegoCosta/trabalhando-remoto/issues/35#issuecomment-832303795>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJ5YVGPI65E4SCZCDAEEG4LTMB4YBANCNFSM4CQ4V3PA>
> .
>
-- 
Danielle Borges

**Comment by wmdebrito** [comment]
> Legal, vou pesquisar a Veem também. Eu presto serviços para uma empresa Americana e tive que enviar o W-8BEN - certificate of Foreign Status for USA tax withholding and reporting, pode ter algo parecido no Canadá.
> On Tue, 4 May 2021 at 20:06 Wagner Brito ***@***.***> wrote: @DanielleBorges <https://github.com/DanielleBorges> @DanielleBorges <https://github.com/DanielleBorges> Conseguiu resolver a tua situacao? Estou na mesma situacao que vc, acabei de voltar para o Brasil e continuo trabalhando remoto e recebendo na minha conta no Canada, precisou abrir PJ? Pode me indicar um contador? Em 2018 eu fui até na receita federal buscando esclarecimentos e ninguém soube me dar uma resposta clara. Não sei se agora com essa onda de trabalho remoto alguma lei mudou. Decidi que era melhor pecar por excesso do que omissão. Abri uma PJ, utilizo o serviço do Bitwage e recebo aqui. Declaro a nota fiscal como exportação de serviços do valor recebido e pago por volta de 4% de imposto. Importante detalhe é que eu faço serviços administrativos, acho que alíquota é menor que desenvolvedores ou algo mais técnico. Fazia tempo que não entrava aqui, vi que tem outros caminhos além da Bitwage. Vou dar uma pesquisada no banco Inter e Payoneer como recomendado acima. Contato do meu contador: Rodrigo Amadio Pacheco Celular. (15)99105-6166 (Claro) ***@***.*** Muito Obrigado pela resposta. Acho que vou por esse caminho também. Eu queria mesmo continuar como PF, eu ainda tenho que ver como vai a questão do imposto no Canadá em relação ao que a minha empresa precisa fazer. Eu trabalho numa empresa que faz transferências também, nosso foco são pequenas e medias empresas. Transferimos para o Brasil. Não é propaganda é só mais uma opção. Pode fazer uma simulação lá. [www.veem.com](http://www.veem.com) — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#35 (comment)](https://github.com/DyegoCosta/trabalhando-remoto/issues/35#issuecomment-832303795)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AJ5YVGPI65E4SCZCDAEEG4LTMB4YBANCNFSM4CQ4V3PA> .
> -- Danielle Borges

Obrigado. Eu preciso ver com a empresa e com revenue do Canada tambem, pois tenho residencia permanent no Canada, mas como já estou há mais de 183 dias not Brasil, já sou considerado residente aqui. Vou tentar ficar como PF, vamos ver se consigo sem pagar muito imposto. Sei que tem uns acordo Brasil x Canada para evitar dupla tributação, mas é dificil achar informações. Estou tentando contato com um especilista na área. Mais está dificil achar, se conseguir eu posto aqui.

**Comment by gustavoramos27** [comment]
> @praguc eu sempre declaro o que recebo. Por exemplo, suponha que num mês eu receba US$ 1.000,00, transfira US$ 800 para a conta no Brasil e use US$ 200 pra gastar no cartão com compras no exterior. Eu declaro os US$ 1.000,00, não os US$ 800.
> 
> Sobre bancos, se vc vai usar a Payoneer não precisa se preocupar com câmbio porque o dinheiro é transferido diretamente em reais pra sua conta. A taxa de câmbio entraria em jogo se vc fosse usar o Itaú pra receber dinheiro, caso em que não precisaria da Payoneer. Mas já que vc vai fazer como pessoa física eu recomendaria o Nubank mesmo (mas vc pode usar qualquer conta que tiver porque a Payoneer transfere pra lá). Transfere dinheiro da Payoneer pro Nubank e declara no carnê-leão o valor que vc recebeu na Payoneer.
> 
> Em todo caso, dá uma lida nesse tópico mesmo e nos outros desse repositório que tem várias alternativas pra receber dinheiro do exterior. Creio que os meios mais utilizados sejam o Bitwage, o Inter e a Payoneer, mas tem outras formas também.
> 
> E só um adendo aqui, é possível que seu primeiro pagamento pela Payoneer de valor mais alto (mais de US$ 500, pelo menos) fique bloqueado. Nesse caso eles vão te pedir mais informações como contrato com a empresa e um site seu (pode ser seu perfil do LinkedIn), vão levar uns dias pra avaliar e depois desbloqueiam. Isso é só uma verificação pra evitar lavagem de dinheiro, novos pagamentos da mesma empresa não serão bloqueados.

Olá Samir,
Também utilizo o Payoneer e pretendo usar o cartão pré-pago para compras aqui no Brasil, mas hoje entrei em contato com uma empresa de contabilidade e me informaram que deixar o dinheiro na conta do exterior seria "sonegação fiscal". Fiquei extremamente confuso, pois para mim não faz sentido e minha intenção é deixar em dólares, mas declarar (sou MEI).
Você conhece algum contador especialista neste assunto? Obrigado!

**Comment by samiraguiar** [comment]
@gustavoramos27 cara, é perfeitamente legal pra uma empresa manter valores em moeda estrangeira, desde que declarados. O que não pode fazer é receber o dinheiro, deixar uma parte lá, e declarar só o que trouxer pra cá, porque aí sim você estaria deixando de pagar imposto devido. Então você declara o que recebeu em moeda estrangeira, sem descontar nada (nem IOF, nem spread, nada), depois se quiser manter lá fora ou trazer pra cá é indiferente. [Nessa discussion](https://github.com/DyegoCosta/trabalhando-remoto/discussions/122) tem mais coisa disso.

Eu confirmei isso com meu contador, que não é especialista, mas tirei a dúvida também com o contador de um amigo, do escritório Manassés Contabilidade Estratégica. Eu não conheço o escritório, mas de acordo com esse amigo, eles têm foco em contabilidade pra área de TI, incluindo os remotos.

Se conseguir mais informações também, ou pontos de atenção, dá um toque aqui por favor.
